aten/src/ATen/native/Normalization.cpp

```diff
@@ -508,7 +524,30 @@ std::tuple<Tensor, Tensor, Tensor> _batch_norm_impl_index_backward(
   const Tensor& save_mean = c10::value_or_else(save_mean_opt, [] {return Tensor();});
   const Tensor& save_var_transform = c10::value_or_else(save_var_transform_opt, [] {return Tensor();});
 
-  if (impl_index == 0) {
+  if (input.numel() == 0) {
+    std::vector<int64_t> dims(input.dim() - 1);
+    dims[0] = 0;
+    std::iota(dims.begin() + 1, dims.end(), 2);
```

Comment:
cool, where is a test that should have been failing previously?

Reply:
https://github.com/pytorch/pytorch/pull/57321/files#diff-4a85700059a0d89291a89a82d2f1ce111293a80c5bc7009b0479fb51b7e5d548R115-R122

I verified the failure before applying the fix.